app/src/androidTest/java/pl/makimo/androidtemplate/ExampleInstrumentedTest.kt

```diff
@@ -3,24 +3,34 @@ package pl.makimo.androidtemplate
 import androidx.test.rule.ActivityTestRule
 
 import org.junit.Test
+import org.junit.Rule
```

Comment:
Have you considered replacing `junit` with [KotlinTest](https://github.com/kotlintest/kotlintest) (it provides PBT for instance)?